LanguageMaster.xml

```diff
@@ -1421,8 +1421,10 @@ can edit in same subtitle file (collaboration)</Information>
     <NewLine>New line (\r\n)</NewLine>
     <NewLineShort>New line (\n)</NewLineShort>
     <AnyDigit>Any digit (\d)</AnyDigit>
+    <NonDigit>Non digit (\D)</NonDigit>
```

Comment:
I would suggest `<NonDigit>Not a digit (\D)</NonDigit>`.

Also, (line 1420)
 `<NonWordBoundary>Non word boundary (\B)</NonWordBoundary>`
 ==> `<NonWordBoundary>Not a word boundary (\B)</NonWordBoundary>`.